Sources/RNLibreToolsiOS13/LibreToolsResources/LibrePro.swift

```diff
@@ -202,7 +226,8 @@ class LibrePro: Sensor {
             if crcReport.contains("FAILED") {
                 if history.count > 0 { // bogus raw data
                     // main?.errorStatus("Error while validating sensor data")
-                    return
+                    logger.error("CRC FAILED")
```

Comment:
I think you should throw here instead as showing error seems a better UX rather than rendering invalid data